pkg/testutil/prometheus.go

```diff
@@ -163,16 +163,16 @@ func (p *Prometheus) Start() error {
 		)
 	}
 	p.addr = fmt.Sprintf("localhost:%d", port)
-	p.cmd = exec.Command(
-		prometheusBin(p.version),
-		append([]string{
-			"--storage.tsdb.path=" + p.db.Dir(),
-			"--web.listen-address=" + p.addr,
-			"--web.route-prefix=" + p.prefix,
-			"--web.enable-admin-api",
-			"--config.file=" + filepath.Join(p.db.Dir(), "prometheus.yml"),
-		}, extra...)...,
-	)
+	args := append([]string{
+		"--storage.tsdb.retention=15d", // Pass retention cause prometheus since 2.8.0 don't show default value for that flags in web/api: https://github.com/prometheus/prometheus/pull/5433
```

Comment:
This is a very minor thing but maybe we should set it to `1d` or `2d`? We don't recommend users to usually have a high on-disk retention since the remote read API is highly inefficient (and because all of the blocks after 2 hours get uploaded either way, and Thanos Store is for that) and it would be nice not to hard-code these parameters which go against those recommendations into the tests. I know that most likely 99% of people won't read them but still it would be nice, I think.

Reply:
I've set it to 2d and change value in the test. Thanks :)